app/com/lucidchart/open/cashy/utils/AssetDataHelper.scala

```diff
@@ -0,0 +1,39 @@
+package com.lucidchart.open.cashy.utils
+
+import java.io._
+import play.api.mvc.MultipartFormData.FilePart
+
+case class AssetNotFoundException(message: String) extends Exception(message)
+
+case class AssetData(
+  bytes: Array[Byte],
+  contentType: Option[String],
+  filename: String
+)
+
+object AssetDataHelper {
+  def getData(file: Option[FilePart[Array[Byte]]], url: Option[String]): AssetData = {
```

Comment:
A better way to do this is an Either[FilePart, String]. It requires that one or the other is specified.